addons/website/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" name="Team Basic">
+    <section class="s_company_team_basic o_colored_level o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10">
+                <div class="o_grid_item g-col-lg-12 g-height-2 col-lg-12" style="grid-area: 1 / 1 / 3 / 13; --grid-item-padding-y: 32px; --grid-item-padding-x: 32px; z-index: 1;">
+                    <h3 class="o_default_snippet_text" style="text-align: center;">Discover our executive team</h3>
+                </div>
+                <div class="o_grid_item g-col-6 g-col-lg-3 g-height-8 col-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 1 / 11 / 4; --grid-item-padding-y: 16px; --grid-item-padding-x: 32px; z-index: 2;">
+                    <p class="o_not_editable" contenteditable="false" style="text-align: center;">
+                        <img alt="" class="o_editable_media img-fluid rounded-circle w-lg-100" src="/web/image/website.s_company_team_image_1" style="text-align: center;"/>
```

Comment:
```suggestion
                        <img alt="" class="o_editable_media img-fluid rounded-circle" src="/web/image/website.s_company_team_image_1" style="width: 100% !important; text-align: center;"/>
```
Same for other images. It doesn't really need responsive sizing IMO as it looks good in full width on mobile as because of the `col-6`.

Also this is not achievable in the editor.